libafl/src/events/llmp.rs

```diff
@@ -340,12 +339,49 @@ where
     }
 }
 
-impl<I, S, SP, ST> EventManager<I, S> for LlmpEventManager<I, S, SP, ST>
+impl<I, OT, S, SP, ST> EventFirer<I, S> for LlmpEventManager<I, OT, S, SP, ST>
```

Comment:
Producer? Firer sounds a bit odd (but since it has a `fire` method I guess it's fine 😁)